pkg/apis/apps/v1/conversion.go

```diff
@@ -23,55 +23,17 @@ import (
 	appsv1 "k8s.io/api/apps/v1"
 	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/kubernetes/pkg/apis/apps"
-	k8s_api_v1 "k8s.io/kubernetes/pkg/apis/core/v1"
 )
 
-func Convert_v1_DeploymentSpec_To_apps_DeploymentSpec(in *appsv1.DeploymentSpec, out *apps.DeploymentSpec, s conversion.Scope) error {
-	if in.Replicas != nil {
-		out.Replicas = *in.Replicas
-	}
-	out.Selector = in.Selector
-	if err := k8s_api_v1.Convert_v1_PodTemplateSpec_To_core_PodTemplateSpec(&in.Template, &out.Template, s); err != nil {
-		return err
-	}
-	if err := Convert_v1_DeploymentStrategy_To_apps_DeploymentStrategy(&in.Strategy, &out.Strategy, s); err != nil {
-		return err
-	}
-	out.RevisionHistoryLimit = in.RevisionHistoryLimit
-	out.MinReadySeconds = in.MinReadySeconds
-	out.Paused = in.Paused
-	if in.ProgressDeadlineSeconds != nil {
-		out.ProgressDeadlineSeconds = new(int32)
-		*out.ProgressDeadlineSeconds = *in.ProgressDeadlineSeconds
-	}
-	return nil
-}
-
 func Convert_apps_DeploymentSpec_To_v1_DeploymentSpec(in *apps.DeploymentSpec, out *appsv1.DeploymentSpec, s conversion.Scope) error {
```

Comment:
if all you do is delegate to the autogenerated method, can you remove the whole method here?

Reply:
I did that initially, but unfortunately I can't - code-generator doesn't generate the public method for it, because auto-generated ones contains a warning.

Reply:
ah, ok... can you add comments to the public ones about why they are required

Reply:
done